### PR TITLE
Updates Invoice docs with new terminology standards

### PR DESCRIPTION
**Pull request by jonikamelcher**

### CODE REVIEW

File: src/api-reference/invoice/v3.payment-request-digest.markdown [hunk 1]
```diff
@@ -5,15 +5,15 @@ redirect_from:
   - /api-reference/invoice/payment-request-digest.html
```
Comment (sarraloew) [inline src/api-reference/invoice/v3.payment-request-digest.markdown:5]: Need to update the title also.

File: src/api-reference/invoice/v3.payment-request.markdown [hunk 2]
```diff
@@ -5,16 +5,16 @@ redirect_from:
   - /api-reference/invoice/payment-request.html
```
Comment (sarraloew) [inline src/api-reference/invoice/v3.payment-request.markdown:5]: Another title update

File: src/api-reference/invoice/v3.payment-request.markdown [hunk 3]
```diff
@@ -53,34 +53,34 @@ PUT /api/v3.0/invoice/paymentrequest
...
+`BuyerCostCenter`|`string`|-|The company defined center responsible for the Invoice.
 `CheckNumber`|`string`|-|Check number of the payment made to the vendor.
 `CompanyBillToAddressCode`|`string`|-|The code which identifies the company location to which the vendor billed the invoice.
 `CompanyShipToAddressCode`|`string`|-|The code which identifies the company location to which the vendor shipped items listed in the invoice.
 `CountryCode`|`string`|-|**Required** The country code.
-`CurrencyCode`|`string`|-|The 3-letter ISO 4217 currency code for the expense report currency. Examples: USD - US dollars; BRL - Brazilian real; CAD - Canadian dollar; CHF - Swiss franc; EUR - Euro; GBO - Pound sterling; HKD - Hong Kong dollar; INR - Indian rupee; MXN - Mexican peso; NOK - Norwegian krone; SEK - Swedish krona.
+`CurrencyCode`|`string`|-|The 3-letter ISO 4217 currency code for the expense report currency. Examples: `USD` - US dollars; `BRL` - Brazilian real; `CAD` - Canadian dollar; `CHF` - Swiss franc; `EUR` - Euro; `GBO` - Pound sterling; `HKD` - Hong Kong dollar; `INR` - Indian rupee; `MXN` - Mexican peso; `NOK` - Norwegian krone; `SEK` - Swedish krona.
```
Comment (sarraloew) [inline src/api-reference/invoice/v3.payment-request.markdown]: should be invoice currency rather than expense report.

**Review by sarraloew — CHANGES_REQUESTED** [review]

**Review by sarraloew — APPROVED** [review]

**Comment by sarraloew** [comment]
:shipit:

**Comment by charlieconcur** [comment]
:shipit:

**Pull request merged**